python/cuml/_thirdparty/sklearn/preprocessing/_function_transformer.py

```diff
@@ -75,13 +75,17 @@ class FunctionTransformer(TransformerMixin, BaseEstimator):
 
     Examples
     --------
-    >>> import numpy as np
-    >>> from sklearn.preprocessing import FunctionTransformer
-    >>> transformer = FunctionTransformer(np.log1p)
-    >>> X = np.array([[0, 1], [2, 3]])
-    >>> transformer.transform(X)
-    array([[0.       , 0.6931...],
-           [1.0986..., 1.3862...]])
+
+    .. code-block:: python
```

Comment:
I think this is code borrowed verbatim from sklearn and hence we usually try not to modify it (makes future updates a PITA)... is there a way to just suppress warnings on this file?

Reply:
I had converted all of the examples to use `.. code-block:: python` thinking it was the cause of all the warnings I was seeing. Turns out, it was something else and we can safely keep them in the same style as sklearn. I'll revert this.

Reply:
Reverted